INSTALL.md

```diff
@@ -141,65 +141,62 @@ packages necessary for building .debs, so be prepared to type your password for
 
-Since https://github.com/ReDetection/homebrew-qtox you can easily install qtox with homebrew 
-```bash
-brew install --HEAD ReDetection/qtox/qtox
-```
+Compiling qTox on OS X for development requires 3 tools, [Xcode](https://developer.apple.com/xcode/) and [Qt 5.4+](http://www.qt.io/qt5-4/), and [homebrew](http://brew.sh).
 
+###Required tools
 
-###OSX Full Install Guide
+First, let's install the dependencies
+* ```brew install git wget```
+* ``git clone https://github.com/tux3/qTox``
+* ```cd qTox```
 
-This guide is intended for people who wish to use an existing or new ProjectTox-Core installation separate to the bundled installation with qTox, if you do not wish to use a separate installation you can skip to the section titled 'Final Steps'.
+###Libraries required to compile
 
-Installation on OSX, isn't quite straight forward, here is a quick guide on how to install;
+Now we are in the qTox folder and need our library dependencies to actually build it.
 
-Note that qTox now requires OpenCV and OpenAL for video and audio.
+I've taken the time to prepare them automatically with our CI system so if you ever have issues redownload them.
 
-The first thing you need to do is install ProjectTox-Core with a/v support. Refer to the INSTALL guide in the PrjectTox-Core github repo.
+* ```wget https://jenkins.libtoxcore.so/job/qTox%20OS%20X/lastSuccessfulBuild/artifact/dep.zip```
+* ```unzip dep.zip```
 
-Next you need to download QtTools (http://qt-project.org/downloads), at the time of writing this is at version .3.0.
-Make sure you deselect all the unnecessary components from the 5.3 checkbox (iOS/Android libs) otherwise you will end up with a very large download.
+As everything from toxcore to filter audio to our special openal fork with echo cancelation is now installed, 
+laugh at some Linux users waiting for a package manager to catch up and figure out what things are called.
 
-Once that is installed you will most likely need to set the path for qmake. To do this, open up terminal and paste in the following;
+###Compiling
 
-```bash
-export PATH=/location/to/qmake/binary:$PATH
-```
+Either open Qt creator and hit build or run qmake && make in your qTox folder and it'll just work™
 
-For myself, the qmake binary was located in /Users/mouseym/Qt/5.3/clang_64/bin/.
+Note that if you use the CLI to build you'll need to add Qt5's bins to your path.
+```export PATH=$PATH:~/Qt/5.4/clang_64/bin/```
 
-This is not a permanent change, it will revert when you close the terminal window, to add it permanently you will need to add echo the above line to your .profile/.bash_profile.
+###Fixing things up
 
-Once this is installed, do the following;
+The bad news is that Qt sucks and breaks our linker paths so we need to fix those.
+First cd in to your qtox.app directory, if you used Qt Creator it's in ```~/build-qtox-Desktop_Qt_5_4_1_clang_64bit-Release``` most likely, otherwise it's in your qTox folder.
```

Comment:
And the job of the doc is to explain and soothe the pain, not to point out just how bad things are. That's what a blog is for.